interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -191,7 +191,8 @@
             curr_month = '0'+curr_month;
           }
           var curr_year = d.getFullYear();
-          $('#dueDate').val(curr_year + "-" + curr_month + "-" + curr_date);
+          var fullDate = curr_year + "-" + curr_month + "-" + curr_date;
+          $('#dueDate').val(moment(fullDate).format(<?php echo js_escape(DateFormatRead('validateJS'))?>));
```

Comment:
neat, I didn't realize we could combine moment with the locale stuff to do this.